tests/openvino/native/quantization/test_quantization_pipeline.py

```diff
@@ -120,3 +122,49 @@ def check_parameters(quantized_model, parameters, path):
     assert quantized_model.has_rt_info(base_path)
 
     check_parameters(quantized_model, quantize_parameters, base_path)
+
+
+@pytest.mark.parametrize(
+    "id, ignored_options",
+    zip(
```

Comment:
You can follow this way of defining parameters for each test case without `zip` and `ids`:
https://github.com/openvinotoolkit/nncf/blob/develop/tests/openvino/native/quantization/test_channel_alignment.py#L67-L74

please also refer to the suggestion about refactoring below, these 2 comments are connected.